### ISSUE TITLE
Ensure Sun's Java Style Checkstyle Coverage

### ISSUE THREAD

**Issue by Ritik2604** [checkstyle/checkstyle#9929]
detected at #6490.
project idea refer: https://github.com/checkstyle/checkstyle/wiki/Checkstyle-GSoC-2021-Project-Ideas#project-name-open-jdk-code-convention-coverage

to update in DOC: https://checkstyle.sourceforge.io/sun_style.html#a4.1
for formatting or standard refer: google java style : https://checkstyle.org/google_style.html
update guideline 1 Introduction - 

link to desired check in sun_checks.xml
test Use src\test\java\com\puppycrawl\tools\checkstyle\checks\sizes\LineLengthCheckTest.java add link of src\it\java\com\sun\checkstyle\test\...
update test files according to sun checks or new JDK guidelines and make unique input files for each test method.
test file should be added in src\it\java\com\sun\checkstyle\test .
update DOC for src/main/resources/sun_checks.xml.
more details for DOC - please read #7698 and see the latest referenced commit to see where and what was changed.
more details for input files - https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-create-Input-files-for-tests

**Comment by Ritik2604** [comment]
@romani Approve this and I am on it.

**Comment by Ritik2604** [comment]
@romani , ~~ping

**Comment by Ritik2604** [comment]
@romani , ~~ping

**Comment by Ritik2604** [comment]
@romani ,~~ping

**Comment by Ritik2604** [comment]
@romani , ~~ping

**Comment by Ritik2604** [comment]
@romani @rnveach Approve this issue.

**Comment by romani** [comment]
> update test files according to sun checks or new JDK guidelines

no updates, we need to create new set of Inputs and test , same as we did for Google style.

**Comment by Ritik2604** [comment]
Yeah I have created new test files and input files I didn't update the existing ones. Please review and suggest any changes
